hw/xbox/dsp/dsp_dma.c

```diff
@@ -54,6 +54,38 @@
 # define DPRINTF(s, ...) do { } while (0)
 #endif
 
+#ifdef DEBUG
```

Comment:
This PR might be a good opportunity to rename the `DEBUG` macros in this file.

`MCPX_APU_DSP_DMA_DEBUG`? A bit of a mouthful, but proper name..

Alternatively, I could use `MCPX_APU_DEBUG` (if we go with a mcpx_apu_debug.h like we did for nv2a_debug.h), and I'll need a bit of grep magic then to split the subcomponents.

Could also use `DSP_DMA_DEBUG` if we decide to isolate the APU / DSP code (as we currently do).

Anyhow, we can also decide this later.

Thoughts?

*(Also: mcpx_apu.c uses `MCPX_DEBUG` when it should be `MCPX_APU_DEBUG` - half the system is in MCPX, so that prefix isn't enough; I don't intend to change that file in this PR though, because I'll touch it later for APU VP anyway)*

Reply:
whats wrong with DEBUG?

Reply:
We've been renaming these. I'm not sure why, but I'm going along with it.
One of the benefits is that we could enable them from the Makefile, by specifying `-DSOMETHING_DEBUG`